CMakeLists.txt

```diff
@@ -89,7 +89,7 @@ option(CIVETWEB_ENABLE_IPV6 "Enables the IP version 6 support" OFF)
 message(STATUS "IP Version 6 - ${CIVETWEB_ENABLE_IPV6}")
 
 # Websocket support
-option(CIVETWEB_ENABLE_WEBSOCKETS "Enable websockets connections" OFF)
+option(CIVETWEB_ENABLE_WEBSOCKETS "Enable websockets connections" ON)
```

Comment:
I don't think it makes sense to change a default setting here. You can always build with `west build ... -- -DCIVETWEB_ENABLE_WEBSOCKETS=ON`, or probably extend your own `CMakeLists.txt`. Changing a default will affect all users, eating up the resources they have.

On the other hand, it's not much, just a few kb of flash. 

I would opt for keeping the default as it is.

If it's not an option, it's better to use 
```
set(CIVETWEB_ENABLE_WEBSOCKETS ON)
```
You can add it in the same file in a Zephyr-specific section:
https://github.com/zephyrproject-rtos/civetweb/blob/f70fa773cdd16868b47c34f1a572ef793d96c65e/CMakeLists.txt#L427-L471

Reply:
Sounds very good to me. Thanks for the advice.

Reply:
Disabled as suggested